src/components/optional-image.jsx

```diff
@@ -3,7 +3,7 @@ import Image from 'react-bootstrap/Image';
 const OptionalImage = ({ image, url, ...rest }) => {
   if (url || image) {
     return (
-      <Image {...rest} src={url || image.url} />
+      <Image {...rest} src={url || image.url} alt={image.alternativeText} />
```

Comment:
You need to null-check this I think. Something like `alt={image && image.alternativeText}`.